styleguide/_themes/derek/scss/snowflakes/stories.scss

```diff
@@ -0,0 +1,94 @@
+.story-colLeft {
+  @include make-md-column(6);
+}
+
+.story-colRight {
+  @include make-md-column(5);
+  @include make-md-column-push(1);
+}
+
+.story-title {
+  text-align: left;
+}
+
+.story-caseStudy-button {
+  @include button;
+  @include button-learning;
+  margin-bottom: 1rem;
+  width: auto;
```

Comment:
This was an exact copy of the buttons we used on events. 
The reason for the `width: auto;` was this: 

<img width="494" alt="screen shot 2016-09-30 at 1 43 17 pm" src="https://cloud.githubusercontent.com/assets/5263371/19002900/e79287a8-8713-11e6-9ab0-b5421fa8a2ce.png">

vs 

<img width="544" alt="screen shot 2016-09-30 at 1 43 30 pm" src="https://cloud.githubusercontent.com/assets/5263371/19002901/e793096c-8713-11e6-8789-257a8810d57f.png">